src/python/pants/backend/python/subsystems/python_native_code.py

```diff
@@ -138,6 +138,25 @@ def check_build_for_current_platform_only(self, targets):
       .format(str(platforms_with_sources)))
 
 
+class BuildSetupRequiresPex(ExecutablePexTool):
+  options_scope = 'build-setup-requires-pex'
+
+  @classmethod
+  def subsystem_dependencies(cls):
+    return super(BuildSetupRequiresPex, cls).subsystem_dependencies() + (PythonSetup.scoped(cls),)
+
+  @property
+  def base_requirements(self):
+    return [
+      PythonRequirement('setuptools'),
+      PythonRequirement('wheel'),
+    ]
+
+  @memoized_property
+  def _python_setup(self):
```

Comment:
Unused - kill.

Reply:
I'm not sure I understand -- `self._python_setup` is [referenced](https://github.com/pantsbuild/pants/blob/268d7d3250e8ff91d1b89296c3cb6d3e428fbeef/src/python/pants/backend/python/subsystems/python_native_code.py#L103) in the `get_targets_by_declared_platform` method?